policyengine_us/variables/gov/states/oh/tax/income/credits/lump_sum_distribution/oh_distribution_credit.py

```diff
@@ -0,0 +1,22 @@
+from policyengine_us.model_api import *
+
+
+class oh_distribution_credit(Variable):
```

Comment:
this should be person level (not sure if just head and spouse or all members)

Reply:
we create `oh_distribution_credit_person` 

Entity = Person; formula also has person in it 

and the `oh_distribution_credit` has just a formula
`adds = ["oh_distribution_credit_person"]`